hackathon_site/event/jinja2/event/landing.html

```diff
@@ -232,11 +232,11 @@ <h3 class="center white-text">FAQ</h3>
     </div>
     <br /><br />
 </div>
-<!--<div class="ombreBackgroundDark2">-->
-<!--    <div class="container section scrollspy" id="sponsors">-->
-<!--        <div class="row">-->
-<!--            <h3 class="center white-text">Sponsors</h3>-->
-<!--            <div class="row sponsorImgs">-->
+<div class="ombreBackgroundDark2">-->
+    <div class="container section scrollspy" id="sponsors">-->
+        <div class="row">-->
+            <h3 class="center white-text">Sponsors</h3>-->
```

Comment:
I commented out "Sponsors" in the header in a previous PR but now that this section is being uncommented, can you also uncomment Sponsors" in the header